EngineBay.DemoModule/DemoModuleEndpoints.cs

```diff
@@ -1,37 +1,45 @@
 namespace EngineBay.DemoModule
 {
-    using System.Security.Claims;
-    using EngineBay.Core;
-
     public static class DemoModuleEndpoints
```

Comment:
Let's see how this file develops, I am almost thinking wee should split this per resource?

I find it work to differentiate between the two base paths, it can only get worse!